File: ru/javastudy/main/Homework1.java
package ru.javastudy.main;

public class Homework1 {

    // задание 1
    private byte a = 127;
    private short s = 32000;
    private int i = 99999999;
    private float f = 5.0F;
    private double d = 15.0D;
    private long l = 1000000000000L;

    private String str = "Hello world!";
    private char c = 'a';
    private boolean bool = false;

    private void greetUser(String name) { // задание 2
        System.out.println("Привет, " + name);
    }

    private boolean isNumberNegative(int num) { // задание 3
        return num < 0;
    }

    private void printNumberSign(int num) { // задание 4
        System.out.println(num > 0 ? num + " положительное число." : num + " отрицательное число.");
    }

    private void printMultipleNumbers() { // задание 5
        for (int i = 1000; i >= 0; i--)
            if (i % 3 == 0) System.out.println(i);
    }

    public static void main(String[] args) {
        Homework1 homework = new Homework1();

        homework.greetUser("Артём");
        System.out.println(homework.isNumberNegative(-15));
        homework.printNumberSign(20);
        homework.printMultipleNumbers();
    }

}

File: main/Homework1.kt
package ru.kotlinstudy.main

// задание 1
val b: Byte = 127
val s: Short = 15000
val i: Int = 500000000
val l: Long = 999999999999999999
val d: Double = 75.0
val f: Float = 15.7F

val bool: Boolean = true
val str: String = "Hello, world"
val c: Char = 'A'

fun main() {
    printMultipleNumbers()
    greet("Артём")
    println(isNumberNegative(2))
    printNumSign(15)
}

// задание 2
fun greet(str: String) {
    println("Привет, $str!")
}

// задание 3
fun isNumberNegative(num: Int): Boolean {
    return num < 0
}

// задание 4
fun printNumSign(num: Int) {
    var state: String = ""

    when {
        num > 0 -> state = "положительное."
        num < 0 -> state = "отрицательное."
    }

    println("Число $num $state")
}

// задание 5
fun printMultipleNumbers() {
    for (i in 1000 downTo 0) {
        if (i % 3 == 0) println(i)
    }
}